declarative/desktop/UserDelegate.qml

```diff
@@ -0,0 +1,229 @@
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * $END_LICENSE$
+ ***************************************************************************/
+
+import QtQuick 2.0
+import QtQuick.Controls 2.0
+import QtQuick.Controls.Material 2.0
+import QtQuick.Layouts 1.0
+import Fluid.Controls 1.0
+
+Card {
+    id: userDelegate
+
+    property string userName: model.userName
+    property string realName: model.realName
+    property string iconFileName: model.iconFileName
+    property string backgroundMode: model.backgroundMode
+    property url pictureUrl: model.pictureUrl
+    property string fillMode: model.fillMode
+    property string primaryColor: model.primaryColor
+    property string secondaryColor: model.secondaryColor
+
+    property bool selected: ListView.isCurrentItem
+
+    property alias field: field
+    property alias busyIndicator: busyIndicator
+
+    signal accepted(string password)
+
+    Material.theme: Material.Light
+    Material.elevation: selected ? 4 : 2
+
+    y: selected ? 0 : (itemSelectedSize.height - itemSize.height) / 2
+    width: selected ? itemSelectedSize.width : itemSize.width
+    height: selected ? itemSelectedSize.height : itemSize.height
+
+    onSelectedChanged: {
+        if (selected)
+            field.forceActiveFocus()
+    }
+
+    Behavior on y {
+        NumberAnimation { duration: Units.shortDuration }
+    }
+
+    Behavior on width {
+        NumberAnimation { duration: Units.shortDuration }
+    }
+
+    Behavior on height {
+        NumberAnimation { duration: Units.shortDuration }
+    }
+
+    QtObject {
+        id: __private
+
+        function activate() {
+            // Select this delegate
+            usersListView.currentIndex = index
+            usersListView.focus = true
+
+            // Give focus to the field anyway, so that if we are
+            // in the lock screen (which has only one delegate) and
+            // the focus is lost for whatever reason the user
+            // will be able to restore it with a click
+            field.forceActiveFocus()
+        }
+    }
+
+    ColumnLayout {
+        anchors {
+            fill: parent
+            margins: Units.largeSpacing
+        }
+
+        CircleImage {
+            id: image
+
+            width: 96
+            height: width
+            visible: status == Image.Ready && String(source).indexOf("sddm/faces/default.face.icon") == -1
+            source: model.iconFileName
+
+            Rectangle {
+                anchors.fill: parent
+                color: "transparent"
+                border.color: Qt.rgba(0, 0, 0, 0.3)
+                radius: width/2
+            }
+
+            MouseArea {
```

Comment:
@iBeliever an alternative would be to have a mousearea on the whole card that doesn't accept buttons but reacts only on hover events and expand the card when the mouse is over - though it might be annoying